asv_bench/benchmarks/package.py

```diff
@@ -0,0 +1,24 @@
+"""
+Benchmarks for pandas at the package-level.
+"""
+import subprocess
+import sys
+
+from pandas.compat import PY37
+
+
+class TimeImport:
+    def time_import(self):
+        if PY37:
+            cmd = [sys.executable, "-X", "importtime", "-c", "import pandas as pd"]
```

Comment:
is there a reason you are not using ``subprocess.run(...).stdout`` ? much simpler

Reply:
force of habit, will change

Reply:
updated